tests/test_query_visitors.py

```diff
@@ -350,6 +350,54 @@
         None,
         id="sequences can mix expressions with literals",
     ),
+    pytest.param(
+        Query("discover", Entity("events"))
+        .set_select(
+            [
+                Column("transaction", output_alias="tn"),
+                Function("count", [], "equation[0]"),
+            ]
+        )
+        .set_groupby(
+            [
+                Column("project_id", output_alias="pi"),
+                Column("transaction", output_alias="tn"),
+            ]
+        )
+        .set_where([Condition(Column("project_id"), Op.IN, (1,))]),
+        (
+            "MATCH (events)",
+            "SELECT transaction AS tn, count() AS equation[0]",
```

Comment:
The column has an alias which is not the same as an output alias but it shows up the same way in the query?

Reply:
Columns do not have aliases in the SDK. That's what this PR is (partially) adding.